cmd/osm-preinstall/osm-preinstall.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+
+	"github.com/openservicemesh/osm/pkg/constants"
+	"github.com/openservicemesh/osm/pkg/logger"
+	"github.com/openservicemesh/osm/pkg/version"
+)
+
+var log = logger.New("osm-preinstall")
+
+func main() {
+	log.Info().Msgf("Starting osm-preinstall %s; %s; %s", version.Version, version.GitCommit, version.BuildDate)
+
+	var verbosity string
+	var enforceSingleMesh bool
+
+	flags := pflag.NewFlagSet("osm-preinstall", pflag.ExitOnError)
+	flags.StringVarP(&verbosity, "verbosity", "v", "info", "Set log verbosity level")
+	flags.BoolVar(&enforceSingleMesh, "enforce-single-mesh", false, "Enforce only deploying one mesh in the cluster")
```

Comment:
```suggestion
	flags.BoolVar(&enforceSingleMesh, "enforce-single-mesh", true, "Enforce only deploying one mesh in the cluster")
```

By default we enforce single mesh in OSM